cilium/network_policy.cc

```diff
@@ -1063,28 +1063,28 @@ class PolicyInstanceImpl : public PolicyInstance {
 // Common base constructor
 // This is used directly for testing with a file-based subscription
 NetworkPolicyMap::NetworkPolicyMap(Server::Configuration::FactoryContext& context)
-    : tls_map_(context.threadLocal()),
-      local_ip_str_(context.localInfo().address()->ip()->addressAsString()),
+    : context_(context.getServerFactoryContext()),
+      tls_map_(context.getServerFactoryContext().threadLocal()),
```

Comment:
missed this: -> `tls_map_(context_.threadLocal())`

Reply:
fixed and thanks :heavy_check_mark: